src/ar_serial_client.c

```diff
@@ -0,0 +1,277 @@
+//--------------------------------------------------------------
+// Author: Dietrich Lachmann
+// Program: ar_serial_client.c
+// Project: AR Structural Testing
+// Affiliation: Oregon State University
+//--------------------------------------------------------------
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <termios.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netdb.h>
+
+#ifndef OPEN_PORT
+#define OPEN_PORT "/dev/ttyUSB0"
+#endif
+
+#ifndef BUFFER
+#define BUFFER 64
```

Comment:
i think this should probably just be 256, since if we have 20 sensors with 6 sig figs each + 19 semicolons that is > 128 bytes. 256 seems fine